charts/kube-prometheus-stack/values.yaml

```
@@ -1033,6 +1033,9 @@ grafana:
       # Allow discovery in all namespaces for dashboards
       searchNamespace: ALL
 
+      # Support for new table panels (in case, grafana auto migrate the table-old panels to table panels)
```

I think the purpose is clearer this way. Do you agree?

```suggestion
      # Support for new table panels, when enabled grafana auto migrates the old table panels to newer table panels
```

---

Thanks!